doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -248,6 +248,10 @@ Bluetooth samples
   * Fixed an issue where the sample would assert with the :kconfig:option:`CONFIG_ASSERT` Kconfig option enabled.
     This was due to calling the :c:func:`bt_iso_chan_send` function from a timer ISR handler and sending SDUs to the controller with invalid timestamps.
 
+* :ref:`direct_test_mode` sample:
+
+   * Added support for the nRF54LM20 PDK.
```

Comment:
I believe you could leave this entry out and add it to https://github.com/nrfconnect/sdk-nrf/pull/22197, instead.